.tekton/basic-component-pipeline.yaml

```diff
@@ -356,6 +356,31 @@ spec:
       operator: in
       values: [ "false" ]
 
+  - name: roxctl-image-scan
+    params:
+    - name: image
+      value: $(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)
+    - name: output_format
+      value: table
+    - name: rox_image
+      value: quay.io/stackrox-io/roxctl:4.4.2
```

Comment:
[nit] It feels weird that the parameter is called `rox_image` and it points to `roxctl` image. If you'll be introducing breaking changes, I suggest renaming that one.